pkg/kn/commands/source/cronjob/describe.go

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cronjob
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"knative.dev/eventing/pkg/apis/sources/v1alpha1"
+	alpha1 "knative.dev/pkg/apis/v1alpha1"
+
+	"knative.dev/client/pkg/kn/commands"
+	"knative.dev/client/pkg/printers"
+)
+
+// Return a new command for describe a cron job source object
+func NewCronJobDescribeCommand(p *commands.KnParams) *cobra.Command {
+
+	cronJobDescribe := &cobra.Command{
+		Use:   "describe NAME",
+		Short: "Describe a CronJob source.",
+		Example: `
+  # Describe a cronjob source with name 'my-cron-trigger'
+  kn source cronjob describe my-cron-trigger`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return errors.New("source name required as argument")
```

Comment:
lets align to other similar messages

Reply:
👍 for now, but we should go over the whole wording anyway. Not sure if repeating the command just called is helpful. But totally agree that it should be consistent, so I updated it here.